test/browser.js

```diff
@@ -0,0 +1,66 @@
+import test from 'ava';
+import createTestServer from 'create-test-server';
+import withPage from './helpers/with-page';
+
+test('prefixUrl option', withPage, async (t, page) => {
+	const server = await createTestServer();
+	server.get('/', (request, response) => {
+		response.end('zebra');
+	});
+	server.get('/api/unicorn', (request, response) => {
+		response.end('rainbow');
+	});
+
+	await page.goto(server.url);
+	await page.addScriptTag({path: './umd.js'});
+
+	await t.throwsAsync(async () => {
+		return page.evaluate(() => {
+			window.ky = window.ky.default;
+			return window.ky('/foo', {prefixUrl: '/'});
+		});
+	}, /`input` must not begin with a slash when using `prefixUrl`/);
+
+	const unprefixed = await page.evaluate(url => {
+		return window.ky(`${url}/api/unicorn`).text();
+	}, server.url);
+	t.is(unprefixed, 'rainbow');
+
+	const prefixed = await page.evaluate(prefixUrl => {
+		return window.ky('api/unicorn', {prefixUrl}).text();
+	}, server.url);
+	t.is(prefixed, 'rainbow');
+
+	await server.close();
+});
+
+test('aborting a request', withPage, async (t, page) => {
+	const server = await createTestServer();
+
+	server.get('/', (request, response) => {
+		response.end('meow');
+	});
+
+	server.get('/test', (request, response) => {
+		setTimeout(() => {
+			response.end('ok');
+		}, 500);
+	});
+
+	await page.goto(server.url);
+	await page.addScriptTag({path: './umd.js'});
+
+	const error = await page.evaluate(url => {
+		return new Promise(resolve => {
+			window.ky = window.ky.default;
+			const controller = new AbortController();
+			const req = window.ky(`${url}/test`, {signal: controller.signal}).text();
+			controller.abort();
+			req.then(resolve).catch(error => resolve(error.toString()));
+		});
```

Comment:
I don't think you need to use the `Promise` constructor here. You can just handle it with async/await and try/catch.

Reply:
I removed `new Promise()` but didn't bother to use `await` and `try` / `catch`. Just seemed more verbose than necessary here. Happy to change it, though, LMK.

Reply:
Looks good